cmd/nerdctl/volume_inspect_test.go

```diff
@@ -51,19 +51,20 @@ func TestVolumeInspectSize(t *testing.T) {
 	base.Cmd("volume", "create", testVolume).AssertOK()
 	defer base.Cmd("volume", "rm", "-f", testVolume).Run()
 
+	var size int64 = 1028
+	createFileWithSize(t, testVolume, size)
 	cmdResult := base.Cmd("volume", "inspect", "--size", testVolume).Run()
 	var dc []native.Volume
-	err := json.Unmarshal([]byte(cmdResult.Stdout()), &dc)
+	var err = json.Unmarshal([]byte(cmdResult.Stdout()), &dc)
```

Comment:
why use var?

Reply:
Can we make `base.InspectVolume` support size?

Reply:
No reason, I added support for size as well.